third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -363,6 +460,15 @@ def to_markdown(
     ):
         """Print DataFrame in Markdown-friendly format.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> df = bpd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+            >>> df.to_string()
+            '   col1  col2\\n0     1     3\\n1     2     4'
```

Comment:
Would it render multiline in the generated doc?

Reply:
AFAIU, doctest test the plain outcome in the cell. When you actually copy and paste the result in a shell, the format will render correctly. Example here:
https://screenshot.googleplex.com/9eSsyGRx43pinSu